test/aircraft/FlightManagementSystem/Fms.spec.js

```diff
@@ -391,6 +391,26 @@ ava('.skipToWaypoint() skips to a waypoint in a different leg', (t) => {
     t.true(fms.currentWaypoint.name === waypointNameMock);
 });
 
+ava('.getNextWaypointModel() returns the `WaypointModel` for the next Waypoint in the collection', (t) => {
```

Comment:
Can we add one more case here where the `currentWaypoint` is the last one in the leg but not the last one in the flightPlan?